src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -107,10 +116,11 @@ const ChatItemMyTextV2 = ({
         message,
         message_type: messageType as MessageType,
         updated_at: time,
-        chatType
+        chatType,
+        attachments
       });
     },
-    [data, username, time, message, messageType, chatType]
+    [data, username, time, message, messageType, chatType, attachments]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` parameter is now included in the `onContextMenuPressed` callback function. Make sure that all places where this callback is used are updated to handle this new parameter.



<!-- This is an auto-generated comment by OSS CodeRabbit -->